.github/CODEOWNERS

```diff
@@ -1,107 +1,596 @@
+
+# ServiceLabel: %Data Factory %Service Attention
+/sdk/datafactory/Microsoft.Azure.Management.DataFactory/            @shawnxzq @lmy269
+
+# ServiceLabel: %Data Lake %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen1 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen2 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Analytics %Service Attention
+/sdk/datalake-analytics/Microsoft.Azure.Management.DataLake.Analytics/            @idear1203
+
+# ServiceLabel: %Data Lake Store %Service Attention
+/sdk/datalake-store/Microsoft.Azure.Management.DataLake.Store/            @sumantmehtams
+
+# ServiceLabel: %Data Migration %Service Attention
+/sdk/datamigration/Microsoft.Azure.Management.DataMigration/            @rgreenMSFT
+
+# ServiceLabel: %Devtestlab %Service Attention
+/sdk/devtestlabs/Microsoft.Azure.Management.DevTestLabs/            @Tanmayeekamath
+
+# ServiceLabel: %Device Provisioning Service %Service Attention
+/sdk/deviceprovisioningservices/Microsoft.Azure.Management.DeviceProvisioningServices/            @nberdy
 
 # PRLabel: %DigitalTwins
-/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @jamdavi
+# ServiceLabel: %Digital Twins %Service Attention
+/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @sourabhguha @inesk-vt @jamdavi
 
+# ServiceLabel: %Event Grid %Service Attention
 # PRLabel: %Event Grid
-/sdk/eventgrid/                                 @Kishp01 @ahamad-MS
+/sdk/eventgrid/                                 @Kishp01 @ahamad-MS @jfggdl @JoshLove-msft
 
 # PRLabel: %Event Hubs
 /sdk/eventhub/                                  @serkantkaraca @jsquire
 
-# PRLabel: %Azure.Identity
-/sdk/identity/                                  @schaabs @alexanderSher
+# ServiceLabel: %Event Hubs %Service Attention
+/sdk/eventhub/Microsoft.*/           @serkantkaraca @samuelkoppes
+
+# ServiceLabel: %Functions %Service Attention
+#/<NotInRepo>/            @ahmedelnably @fabiocav
+
+# ServiceLabel: %Graph.Microsoft %Service Attention
+#/<NotInRepo>/            @dkershaw10 @baywet
+
+# ServiceLabel: %Guest Configuration %Service Attention
+/sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/            @mgreenegit @vivlingaiah
+
+# ServiceLabel: %HDInsight %Service Attention
+/sdk/hdinsight/            @aim-for-better @idear1203 @deshriva
+
+# ServiceLabel: %HPC Cache %Service Attention
+#/<NotInRepo>/            @romahamu @omzevall
+
+# ServiceLabel: %Import Export %Service Attention
+#/<NotInRepo>/            @madhurinms
 
 # PRLabel: %Iot
 /sdk/iot/                                       @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @prmathur-microsoft @bikamani @barustum @jamdavi
```

Comment:
you can remove @prmathur-microsoft on this, if you like. She has moved on to another team.

Reply:
Done.